### DISCUSSION TITLE
TOC with more than two levels

### DISCUSSION THREAD

**Discussion by eduardomartini** [alshedivat/al-folio#2836] [Q&A]
Hi,

I am toying with the TOC, and it works out of the box, except that I don't manage it to show more than two levels.

I have a page as simple as 

> ...
> toc:
>   sidebar: left
> ---
> Level 1 
> ## Level 2.1 
> 
>  ### Level 3.1
>  #### Level 4.1
>  #### Level 4.2
>  ### Level 3.2 
>  #### Level 4.1
>  #### Level 4.2
>  ## Level 2.2 
>  # Level 1.2 
> 

I added 

> toc:
>   h_min: 1 
>   h_min: 5 
>   min_level: 1 
>   max_level: 5 

to both the page header and to _config, but that didn't change anything. Am I missing something?

**Comment by george-gca** [comment]
I just tested on the sample post and it worked without adding anything to the page header or the _config.yml:

![image](https://github.com/user-attachments/assets/121ffc90-1b3a-478a-a812-ca20d1efb912)

It doesn't even have to have text after the headers, as I was suspecting.

![image](https://github.com/user-attachments/assets/8f5527bd-6d64-4e0e-a31b-ac543cee6dc4)

**Comment by george-gca** [comment]
Sorry, only now I realized that you were referring to the toc in sidebar. For this we use [bootstrap-toc](https://afeld.github.io/bootstrap-toc/) underneath. It seems to be a limitation of this library. They even have a link in their docs to:

[Adding links to all headings on your page](http://bryanbraun.github.io/anchorjs/)
